config/routes.rb

```diff
@@ -2,7 +2,7 @@
   root "welcome#index"
 
   get "/auth/google_oauth2/callback", to: "sessions#create"
-
+  delete '/sessions', to: 'sessions#destroy'
```

Comment:
we had to handroll this, as it was not exposed by `resources` the way we needed it to be.